src/viam/logging.py

```diff
@@ -44,6 +45,8 @@ def getLogger(name: str) -> logging.Logger:
 
 
 def addHandlers(logger: logging.Logger):
+    logger.handlers.clear()
```

Comment:
This is to fix an issue where logs were getting doubled if the level was changed